source/_integrations/shelly.markdown

```diff
@@ -127,6 +127,14 @@ The integration uses the following strategy to name its entities:
 - If `Channel Name` is set in the device, the integration will use it to generate the entities' name, e.g. `Kitchen Light`
 - If `Channel Name` is set to the default value, the integration will use the `Device ID` and default channel name to generate the entities' name, e.g. `ShellyPro4PM-9808D1D8B912 switch_0`.
 
+## Cover entities
+
+Shelly 2PM Gen3 supports `tilt` for `cover` entities. To enable this feature, you need to configure the device:
```

Comment:
According to Shelly docs this is supported for:
- Shelly 2PM Gen3
- Shelly Pro 2PM
- Shelly Pro Dual Cover PM

Reply:
Yes, but we don't know when this feature will be available for Pro 2PM and Pro Dual Cover PM. Currently, Pro Dual Cover PM with firmware 1.4.2 doesn't support tilt.

Reply:
If it is not supported we should not mention it 👍